schema.sql

```diff
@@ -1,5 +1,8 @@
 /* Database schema to keep the structure of entire database. */
 
 CREATE TABLE animals (
-    name varchar(100)
-);
+    name varchar(100),
```

Comment:
- [ ] According to the requirements below, the animal table should have the following columns but you are missing the `id` column. It is important to have an `id` for easier distinction of animals. 
```
Create a table named animals with the following columns:
id: integer
name: string
date_of_birth: date
escape_attempts: integer
neutered: boolean
weight_kg: decimal
```